ldms/src/sampler/ibmad_records_sampler/ibmad_records_sampler.c

```diff
@@ -764,6 +788,23 @@ static int config(struct ldmsd_plugin *self,
                 return rc;
         }
 
+        value = av_value(avl, "refresh_interval_sec");
+        if (value != NULL) {
+                char *end;
+                long val;
+
+                strip_whitespace(&value);
```

Comment:
the strip_whitespace call (and the bit called) should be entirely unnecessary, as the av processing should be delivering values that are already stripped (lib/src/ovis_util/util.c:tokenize:strtok_r(... " \t\n"...). 
Did you find a case where something was not stripped? If so we should fix tokenize.

Reply:
@morrone, @baallan what is the status of this pull request?

Reply:
If @morrone is busy but will fix the nits soon post-release, we can call it good enough for release.

Reply:
Yeah, I probably won't get to it. And the same nits are elsewhere in the code already (they were literal cut-and-pastes from elsewhere).